extra_docs/quickstart.md

```diff
@@ -28,28 +28,41 @@ Please keep in mind that this quick start guide uses Javascript 2017 syntax.
 
 The examples are made with a simple data schema in mind: think a basic social media platform where `users` can write `posts` and/or `comments` to `posts`. A `post` can be related to any number of `comments`, while each `post` and each `comment` has a `user` as author.
 
-### Importing the SDK into your JS project
+### [Importing the SDK into your JS project](#importing-sdk)
```

Comment:
If this is to create anchors, I think Markdown doesn't support it. This makes links _to_ the anchors (which don't exist)

Instead Markdown/GitHub automatically creates anchors for each header, by using something like this:
* lowercase
* replace space with -

You can see/check in preview of the file

Reply:
in this case this markdown file is used by `compodoc` to generate the docs, for some reason it doesn't create the anchors so I needed to force (you can see here the current rendered version: https://jexia.github.io/jexia-sdk-js/additional-documentation/quick-start-guide.html)

Reply:
The anchors are not working in your example: 
```
<h3 id="importing-the-sdk-into-your-js-project">Importing the SDK into your JS project</h3>
```
or it is because it is not updated yet?

Reply:
oh it's because it's not updated yet